packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/Tooltip.module.scss

```diff
@@ -1,21 +1,59 @@
 .flamegraphTooltip {
-  max-width: 80%;
+  width: 320px;
 
   position: fixed;
   background: #ffffff;
-  white-space: nowrap;
   box-shadow: 1px 2px 4px 0px rgba(0, 0, 0, 0.3);
   border-radius: 4px;
-  padding: 3px 5px;
   color: #333;
   font-size: 12px;
   visibility: hidden;
-  z-index: 1;
-}
+  z-index: 2;
+
+  .flamegraphTooltipName {
+    border-bottom: 1px solid #333;
+    font-weight: bold;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    padding: 8px 5px;
+  }
+
+  .functionName {
+    padding: 8px 5px 0;
+    min-height: 50px;
```

Comment:
50px = ~2 lines

https://user-images.githubusercontent.com/47758224/178529516-893de6c1-8d7c-4064-9400-41e37bee5ff2.mov

Reply:
I don't quite get this part, why minimum 2 lines?

Reply:
i wanted to ask what you do think about it ? ....ref to [this](https://github.com/pyroscope-io/pyroscope/pull/1208#issuecomment-1175155291)...or this min-height doesnt make any sense because mostly we have long function names

Reply:
yeah for this use case I don't think we need the minimum 2 lines , but I like the idea :)